c/include/libsbp/cpp/state.h

```diff
@@ -85,6 +86,11 @@ class State {
     // NOLINTNEXTLINE
     return sbp_process_payload(&state_, sender_id, msg_type, msg_length, const_cast<u8 *>(payload));
   }
+
+  template<typename T>
+  s8 send_message(u16 sender_id, const T &msg) {
```

Comment:
This somehow got missed from the initial import of the prototype generator. It's used heavily in the generate C++ tests for the new API